src/canto/client.py

```diff
@@ -24,6 +25,10 @@ class ResourceNotFound(ApiException):
     pass
 
 
+class AccessTokenRequired(ApiException):
```

Comment:
not sure if this should be api exception but don't really know what it should be, so whatever

Reply:
ok